plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2297,7 +2297,9 @@ elementgui.living_entity.immune_trident=Trident
 elementgui.living_entity.immune_anvil=Anvil
 elementgui.living_entity.immune_wither=Wither
 elementgui.living_entity.immune_dragon_breath=Dragon Breath
-elementgui.living_entity.is_water_mob=Is water entity (breathes underwater, is not pushed by water flow)
+elementgui.living_entity.is_water_mob=Is water entity (water type navigator and motion controller)
```

Comment:
This parameter also allows entity to spawn in fluid which is not the case by default. Since this is a lot of text, should be moved to help tips likely

Reply:
Or `<br><small>` maybe?

Reply:
Possibly if it is not too long visually